docs/grpc/c#.md

```diff
@@ -0,0 +1,217 @@
+# How to write a C# gRPC client for the Lightning Network Daemon
+
+This section enumerates what you need to do to write a client that communicates with `lnd` in C#.
+
+
+### Prerequisites
+
+* A unix terminal for Windows - this tutorial uses [Cygwin](https://www.cygwin.com/)
+
+
+### Setup and Installation
+
+`lnd` uses the `gRPC` protocol for communication with clients like `lncli`.
+
+`gRPC` is based on protocol buffers, and as such, you will need to compile the `lnd` proto file in C# before you can use it to communicate with `lnd`.
+
+This assumes you are using a Windows machine, but it applies equally to Mac and Linux.
+
+* Open Visual Studio, and create a new `.net core` console application called `lnrpc` at the root directory (Windows : `C:/`)
```

Comment:
Now that you switched to `.net core`, you could just as well list the `dotnet` cli commands here. That way visual studio is not a requirement anymore, allowing the steps to be followed without improvisation on mac/linux. 
```
mkdir lndclient
cd lndclient
dotnet new console
dotnet add package Grpc.Tools --version 1.17.0
```